### DISCUSSION TITLE
Multiple button toolbars in the "toolbars" section?

### DISCUSSION THREAD

**Discussion by krisu5** [reupen/columns_ui#403] [Q&A] [answered]
Is it possible to have two or more "button toolbars" in the toolbars section? I couldn't found a way.

_Example where I want another set of buttons (blue area):_

![Näyttökuva 2021-11-17 191309](https://user-images.githubusercontent.com/3540275/142249039-bff99d04-6200-4f48-add7-a3799d40fbaf.png)

**Comment by reupen** [comment, marked as answer]
Hi,

Yes, it is. From the context menu, hold down Shift when selecting Toolbars/Buttons and it'll add a second one.

**Reply by krisu5** [reply]
Thank you!
